packages/vitest/src/types/config.ts

```diff
@@ -198,11 +198,18 @@ export interface InlineConfig {
   globalSetup?: string | string[]
 
   /**
-   * Pattern of file paths to be ignore from triggering watch rerun
+   * Glob pattern of file paths to be ignore from triggering watch rerun
+   */
+  watchExclude?: string[]
+
+  /**
+   * Glob patter of file paths that will trigger the whole suite rerun
    *
-   * @default [/\/node_modules\//, /\/dist\//]
+   * Useful if you are testing calling CLI commands
+   *
+   * @default []
    */
-  watchIgnore?: (string | RegExp)[]
+  dumbWatchInclude?: string[]
```

Comment:
Emm, maybe:

- `forceRerunTargets`
- `forceRerunTriggers`
- `forceRerunEntries`

Reply:
I want to be consistent with "exclude/include"